Source/WebCore/PAL/pal/spi/cocoa/PassKitSPI.h

```diff
@@ -549,6 +536,24 @@ NS_ASSUME_NONNULL_END
 
 #endif
 
+NS_ASSUME_NONNULL_BEGIN
+
+#if HAVE(PASSKIT_DEFAULT_SHIPPING_METHOD) && !USE(APPLE_INTERNAL_SDK)
+@interface PKShippingMethods : NSObject
+- (instancetype)initWithMethods:(NSArray<PKShippingMethod *> *)methods defaultMethod:(nullable PKShippingMethod *)defaultMethod;
+@end
+
+@interface PKPaymentRequest ()
+@property (nonatomic, copy) PKShippingMethods *availableShippingMethods;
+@end
+
+@interface PKPaymentRequestUpdate ()
+@property (nonatomic, copy) PKShippingMethods *availableShippingMethods;
+@end
+#endif
```

Comment:
NIT: I'd add a `// HAVE(PASSKIT_DEFAULT_SHIPPING_METHOD) && !USE(APPLE_INTERNAL_SDK)` here so that it's clear what this is ending.